src/helpers/util/combinePositionAndSize.js

```diff
@@ -0,0 +1,22 @@
+//@flow
+import Style from "further";
+
+import createSizeHelpers from "./createSizeHelpers";
+import createPositionalHelpers from "./createPositionalHelpers";
+import combine from "./combine";
+
+export default (
```

Comment:
This is just composition of `createHelpers + a reducer`, could we make this more universal or clean it up so that it can be used for things beyond margin / padding? Or if it is just composition, is it even needed after we refactor to `createHelpers`?

It may be?